pkg/affected_apps/config_matcher.go

```diff
@@ -40,9 +41,18 @@ func (b *ConfigMatcher) AffectedApps(ctx context.Context, changeList []string, t
 		appSetList = append(appSetList, ApplicationSet{appset.Name})
 	}
 
-	var appsSlice []config.ApplicationStub
-	for name, appPath := range appsMap {
-		appsSlice = append(appsSlice, config.ApplicationStub{Name: name, Path: appPath})
+	argoApps, err := b.argoClient.GetApplications(ctx)
+	if err != nil {
+		return AffectedItems{}, errors.Wrap(err, "failed to list applications")
+	}
+
+	var appsSlice []v1alpha1.Application
+	for _, app := range argoApps.Items {
+		if _, ok := appsMap[app.Name]; !ok {
```

Comment:
Should we log failures here under a debug log?

Reply:
This checks to see if an argo app was one of the triggered apps. It's not a failure, it just means "this app wasn't changed in this pull request". I'll change the variable names to make that obvious.